pyproject.toml

```diff
@@ -0,0 +1,18 @@
+[tool.poetry]
+name = "back2school"
+version = "0.1.0"
+description = "Guided lessons on Python using LLMs"
+authors = ["Vincent Chang and Reuben Brasher"]
```

Comment:
Please include my email address:  vincent.chang167@gmail.com

Reply:
That is a good idea. Done.